ts/client/src/accounts/group.ts

```diff
@@ -359,7 +359,7 @@ export class Group {
         Array.from(this.serum3MarketsMapByExternal.values()).map(
           (serum3Market) =>
             Market.load(
-              client.program.provider.connection,
+              client.program.provider.connection as any,
```

Comment:
this is probably masking an underlying version incompatibility issue, would it be possible to just update or override the dependency?